### PR TITLE
feat: add env variable for sparrow github and login flow style 

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
feat: add env variable for sparrow github and login flow style

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,7 @@
 VITE_API_URL="http://localhost:9000"
+VITE_API_TIMEOUT="5000"
 VITE_ENABLE_MIX_PANEL="false"
 VITE_MIX_PANEL_TOKEN=""
 VITE_SPARROW_SUPPORT_EMAIL="support@example.dev"
-VITE_AUTH_URL="http://localhost:1421"
\ No newline at end of file
+VITE_AUTH_URL="http://localhost:1421"
+VITE_SPARROW_GITHUB="https://github.com"
```
[the remark this answers is not in the document]
Comment (itsmdasifraza) [reply .env.example]: resolved

**Review by parthdalaltech — APPROVED** [review]

**Pull request merged**